doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -1,37 +1,42 @@
 = Cluster Resources =
 
-== What is a Cluster Resource ==
+== What is a Cluster Resource? ==
 
 indexterm:[Resource]
 
-The role of a resource agent is to abstract the service it provides
-and present a consistent view to the cluster, which allows the cluster
-to be agnostic about the resources it manages.
+A resource is a service made highly available by a cluster.
+The simplest type of resource, a 'primitive' resource, is described
+in this chapter. More complex forms, such as groups and clones,
+are described in later chapters.
 
+Every primitive resource has a 'resource agent'. A resource agent is an
+external program that abstracts the service it provides and present a
+consistent view to the cluster.
+
+This allows the cluster to be agnostic about the resources it manages.
 The cluster doesn't need to understand how the resource works because
 it relies on the resource agent to do the right thing when given a
-+start+, +stop+ or +monitor+ command.
-
-For this reason it is crucial that resource agents are well tested.
+`start`, `stop` or `monitor` command. For this reason, it is crucial that
+resource agents are well-tested.
 
-Typically resource agents come in the form of shell scripts, however
+Typically, resource agents come in the form of shell scripts. However,
 they can be written using any technology (such as C, Python or Perl)
 that the author is comfortable with.
 
 [[s-resource-supported]]
-== Supported Resource Classes ==
+== Resource Classes ==
 
 indexterm:[Resource,class]
 
-There are six classes of agents supported by Pacemaker:
+Pacemaker supports six classes of agents:
 
 * OCF
-* LSB
-* Upstart
-* Systemd
-* Fencing
 * Service
-* Nagios
+** LSB
+** Upstart
+** Systemd
```

Comment:
On 03/02/2015 05:17 PM, Andrew Beekhof wrote:
>>  * Service
>> -* Nagios
>> +** LSB
>> +** Upstart
>> +** Systemd
> 
> Why are these listed as sub-points of OCF?
> They're not related

They're sub-points of service, not OCF. See new:

http://people.redhat.com/kgaillot/doc/en-US/Pacemaker/1.1-pcs/html/Pacemaker_Explained/s-resource-supported.html


I sub-listed them b/c previously it said "There are six classes" and
then listed seven bullets. "Seven" would not make sense either since
"service" is not really a separate class. See old:

http://clusterlabs.org/doc/en-US/Pacemaker/1.1-pcs/html/Pacemaker_Explained/s-resource-supported.html